ensembl-track-api.openapi.yaml

```
@@ -89,4 +113,24 @@ components:
           example: GREY
         additional_info:
           type: string
-          example: Forward strand
\ No newline at end of file
+          example: Forward strand
+        description:
+          type: string
+          example: Shows all protein coding genes on the forward strand.\nPart of the GENCODE Comprehensive gene set
+        sources:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                type: string
+                example: GENCODE
+              url:
+                type: string
+                example: https://gencodegenes.org
+        rank:
+          type: number
+          example: 2
+        is_visible:
```

There is a [proposition](https://genomes-ebi.slack.com/archives/CC2RKR525/p1692875598494129?thread_ts=1692870200.220029&cid=CC2RKR525) to also add a field for datafiles location. For example:
```suggestion
        data_files:
          type: object
          properties:
            bigbed:
              type: string
              example: variant-labels-dbsnp.bb
            bigwig:
              type: string
              example: variant-summary-dbsnp.bw
        is_visible:
```
But in this schema the key names would be different depending on the type of track so it might need some more thinking.

---

Do we actually need/want to make this changes to the track api itself during this sprint? I.e. are we expecting by the end of this sprint the track api to report correct file locations? If not, I would argue for doing this separately, in a different PR.

---

The GB integration is for a future sprint so yes, let's drop it for now.